### PR TITLE
Update Falco links and details

### PR DESCRIPTION
**Pull request by fntlnz**
Hi @walidshaari ! Thanks for putting this together.

This is very useful!

This patch adds some more resources for Falco and does change to follow the [brand guidelines](https://github.com/falcosecurity/falco/tree/master/brand) for naming .

Thanks again,
Lore
Signed-off-by: Lorenzo Fontana <lo@linux.com>

### CODE REVIEW

**Comment by walidshaari** [comment]
My day is just getting better, thank you 
for the new lines 190-193 with 3rd part resources, we need usually :triangular_flag_on_post: to denote that these links cannot be referenced in the exam open tab.

@myugan  what do you think? can you please review it

**Comment by walidshaari** [comment]
Something similar to the aqua external resources in the below image using `:triangular_flag_on_post: `

image

**Comment by myugan** [comment]
Agreed @walidshaari for CVE it will be better to put it on CVE section separate references because this is related to `update kubernetes frequently` in the syllabus to prevent our cluster vulnerable to attack by internal/external.

File: README.md [hunk 1]
```diff
@@ -186,15 +186,19 @@ Offical exam objectives you review and understand in order to pass the test.
 
 
 1. Perform behavioural analytics of syscall process and file activities at the host and container level to detect malicious activities
-  - [Old kubernetes.io URL: install falco on k8s 1.17](https://v1-17.docs.kubernetes.io/docs/tasks/debug-application-cluster/falco/)
+  - [Old kubernetes.io URL: install Falco on k8s 1.17](https://v1-17.docs.kubernetes.io/docs/tasks/debug-application-cluster/falco/)
```
Comment (walidshaari) [inline README.md:189]: needs `:triangular_flag_on_post:` after the dash before the square bracket, same should be applied to any external resources that are not kubernetes.io/{docs|blog} or github.com/kubernetes

**Comment by walidshaari** [comment]
Github now allows us to modify commit code, I will see if I can add these flags myself

**Review by walidshaari — APPROVED** [review]
LGTM after adding the flags

**Review by myugan — APPROVED** [review]

**Pull request merged**